shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -16,45 +18,79 @@
  * Results for one fault computation.
  *
  * @author Boubakeur Brahimi
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
  */
 public final class FaultResult extends AbstractExtendable<FaultResult> {
 
-    private final String id;
+    private final Fault fault;
 
-    private final double threePhaseFaultCurrent;
+    private final double threePhaseFaultActivePower;
 
-    private final List<FeederResult> feederResults; // optional
+    private final Duration timeConstant;
```

Comment:
What is the meaning/purpose of this attribute?
Is it something that could really be filled by a static simulator?